pkg/controller/certificates/issuing/issuing_controller.go

```diff
@@ -130,7 +139,8 @@ func NewController(
 		client:                   client,
 		recorder:                 recorder,
 		clock:                    clock,
-		secretsManager:           secretsManager,
+		secretsUpdateData:        secretsManager.UpdateData,
+		fieldManager:             util.PrefixFromUserAgent(restConfig.UserAgent),
```

Comment:
Does this line suggest that every controller will share the same `fieldManager`?

Reply:
For now, yes. Please see my other comment.

Reply:
What _is_ the UserAgent we're going to be using here? A new issuing controller specific name? (not sure what other comment you mean - sorry if it's already answered!)

Reply:
No fieldmanager/user agent is being changed in this PR, and am leaving it for a followup PR (where the certificate controllers will all be changed to use Apply).

I'm currently in the process of writing a small design doc for why we want to use apply, and the user agents + field managers we will be using (or at least the naming scheme).

Reply:
But right now, the user agent is `controller/v0` (something like that), and field manager is `controller`.